tools/hooks/scary_changes.rb

```diff
@@ -72,6 +72,20 @@ def detect_new_table_or_new_column
     EOS
   end
 
+  def detect_column_rename
+    changes = @all.grep(/^dashboard\/db\/migrate\//)
+    return if changes.empty? || !@changed_lines.include?("rename_column")
```

Comment:
@sureshc I saw your comment in the doc about displaying this for all migrations. I opted to only have this run for `rename_column` and not all migrations added to avoid warning fatigue. I find that if I see a warning enough times that doesn't apply to my changes then I tend to start ignoring it. But I'm open to showing it for every migration if you think it's really worth it.

Reply:
Makes sense. Could we have a dedicated check for `add_column`, `add_index`, and `change_column` warning that a migration on a large table (>10M rows) needs to be reviewed and tested with Infrastructure cabal to avoid negatively impacting production database performance because they may cause MySQL to rebuild the entire table? Including a link to this documentation might help: `https://dev.mysql.com/doc/refman/5.7/en/innodb-online-ddl-operations.html#online-ddl-column-operations`

Reply:
I added another check, let me know if you'd like it changed at all!

Reply:
Awesome. Thank you!